analyses/fusion_filtering/02-fusion-filtering.R

```diff
@@ -63,17 +68,24 @@ option_list <- list(
 
 opt <- parse_args(OptionParser(option_list=option_list))
 
-standardFusionFile<-opt$standardFusionFile
+# TO-DO
+# multiple opt values for each caller and output name from input fusion calls and expression Matrix?
+standardFusionFileSTARFusion<-opt$standardFusionFileSTARFusion
+standardFusionFileArriba<-opt$standardFusionFileArriba
 expressionMatrix<-opt$expressionMatrix
-reathroughFilter<-opt$reathroughFilter
+readthroughFilter<-opt$readthroughFilter
 expressionFilter<-opt$expressionFilter
 artifactFilter<-opt$artifactFilter
 junctionReadCountFilter<-opt$junctionReadCountFilter
 spanningFragCountFilter<-opt$spanningFragCountFilter
 referenceFolder<-opt$referenceFolder
 readingFrameFilter<-opt$readingFrameFilter
 
-standardFusioncalls<-readr::read_tsv(standardFusionFile)
+standardFusioncallsSTARFusion<-readr::read_tsv(standardFusionFileSTARFusion)
```

Comment:
I would approach this a bit differently. Imagine you end up with 5 callers. With this approach, you'd have to include 5 separate arguments. Instead you could take a comma-separated list of values as an argument (say `--input_files`) and parse it. 

Here's an example from @cansavvy where she has an argument that can take comma-separated values:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/262d15af567e6b1e6a5afa5a9aa152b556d7a1a1/analyses/snv-callers/scripts/02-run_eval.R#L79 

It's parsed here (you shouldn't need the `tolower` bit): https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/262d15af567e6b1e6a5afa5a9aa152b556d7a1a1/analyses/snv-callers/scripts/02-run_eval.R#L193

Then, for each filename in the resulting vector you would read it in as a `data.frame`. My inclination is to store these as a `list` of `data.frame` probably by `lapply`-ing `read_tsv` to the vector of filenames. `dplyr::bind_rows()` should accept a list of `data.frame`.

Reply:
done!